Gemfile

```diff
@@ -146,7 +146,7 @@ group :test do
   gem 'capybara', '~> 2.3.0'
   gem 'capybara-screenshot'
   gem 'selenium-webdriver', '~> 2.43.0'
-  gem 'timecop', "~> 0.6.1"
+  gem 'timecop'
```

Comment:
Because people tend to use "bundle update" instead of "bundle (install)", we tend to restrict the versions.